lib/Query/Query.php

```diff
@@ -30,14 +30,42 @@ class Query
     const HYDRATE_DOCUMENT = 'document';
     const HYDRATE_PHPCR = 'phpcr_node';
 
-    private $hydrationMode;
-    private $locale;
+    /**
+     * @var QueryInterface
+     */
+    private $phpcrQuery;
+
+    /**
+     * @var EventDispatcherInterface
+     */
     private $dispatcher;
+
+    /**
+     * @var null|string
+     */
     private $primarySelector;
-    private $phpcrQuery;
+
+    /**
+     * @var null|string
+     */
+    private $locale;
+
+    /**
+     * @var int
+     */
     private $maxResults;
+
+    /**
+     * @var int
+     */
     private $firstResult;
```

Comment:
I would say leave it as it is. It is also named this way in Doctrine ORM, so changing it would just be more cognitive overhead.